enterprise/payara.eecommon/nbproject/org-netbeans-modules-payara-eecommon.sig

```diff
@@ -1231,16 +1231,16 @@ supr java.lang.Enum<org.netbeans.modules.payara.eecommon.api.config.JavaEEModule
 hfds LOGGER,j2eeModuleTypeToValue,moduleType
 
 CLSS public abstract org.netbeans.modules.payara.eecommon.api.config.PayaraConfiguration
-cons protected init(org.netbeans.modules.j2ee.deployment.devmodules.api.J2eeModule,org.netbeans.modules.payara.eecommon.api.config.J2eeModuleHelper,org.netbeans.modules.payara.tooling.data.PayaraVersion) throws org.netbeans.modules.j2ee.deployment.common.api.ConfigurationException
-cons protected init(org.netbeans.modules.j2ee.deployment.devmodules.api.J2eeModule,org.netbeans.modules.payara.tooling.data.PayaraVersion) throws org.netbeans.modules.j2ee.deployment.common.api.ConfigurationException
+cons protected init(org.netbeans.modules.j2ee.deployment.devmodules.api.J2eeModule,org.netbeans.modules.payara.eecommon.api.config.J2eeModuleHelper,org.netbeans.modules.payara.tooling.data.PayaraPlatformVersionAPI) throws org.netbeans.modules.j2ee.deployment.common.api.ConfigurationException
+cons protected init(org.netbeans.modules.j2ee.deployment.devmodules.api.J2eeModule,org.netbeans.modules.payara.tooling.data.PayaraPlatformVersionAPI) throws org.netbeans.modules.j2ee.deployment.common.api.ConfigurationException
 cons public init()
  anno 0 java.lang.Deprecated()
 fld protected final java.io.File primaryDD
 fld protected final java.io.File secondaryDD
 fld protected final org.netbeans.modules.j2ee.deployment.devmodules.api.J2eeModule module
 fld protected final org.netbeans.modules.payara.eecommon.api.config.J2eeModuleHelper moduleHelper
 fld protected org.netbeans.modules.payara.eecommon.api.config.DescriptorListener descriptorListener
-fld protected org.netbeans.modules.payara.tooling.data.PayaraVersion version
+fld protected org.netbeans.modules.payara.tooling.data.PayaraPlatformVersionAPI version
```

Comment:
Keep the original file type. Make the new one (package) `private`!? Encapsulation of fields is always better API choice.